ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -234,6 +231,28 @@ def global_step_transform(*args, **kwargs):
                 global_step_transform=global_step_transform
             )
 
+        Another example where the State Attributes ``trainer.state.alpha`` and ``trainer.state.beta``
+        are also logged along with the NLL and Accuracy after each iteration:
+        
+        .. code-block:: python
+
+            tb_logger.attach(
```

Comment:
Adding lines like that in the code is not good neither. Check here: https://deploy-preview-2137--pytorch-ignite-preview.netlify.app/generated/ignite.contrib.handlers.tensorboard_logger.html#ignite.contrib.handlers.tensorboard_logger.OutputHandler

Reply:
Okay, I'll fix the flake8 issue and try with no blank lines.